.devcontainer/socket_proxy.py

```
@@ -0,0 +1,100 @@
+import signal
+import ctypes
+
+def pipe(source, destination):
+    try:
+        while True:
+            data = source.recv(4096)
+            if not data:
+                break
+            destination.sendall(data)
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)